tests/integration/test_get_insert_size_metrics.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from pathlib import Path
+
+
+@pytest.mark.workflow(name='test_get_insert_size_metrics')
```

Comment:
The idea is the name of test `test_get_insert_size_metrics` and what it is asserting `metrics_match`. So `test_get_insert_size_metrics_match` -> `test_get_insert_size_metrics_metrics_match`